From fe5f9fb63427d68cccaef1b20ae29e582365e955 Mon Sep 17 00:00:00 2001
From: Pedro Palazon Candel <pedro@joyent.com>
Date: Tue, 16 Oct 2018 19:20:20 +0200
Subject: [PATCH] TRITON-769 wf-runner is crashing in metrics calc on jobs that
 fail with "workflow timeout" Reviewed by: Mike Gerdts
 <mike.gerdts@joyent.com> Approved by: Mike Gerdts <mike.gerdts@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 4305e6d..7606314 100644
--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
     "uuid": "3.0.1",
     "vasync": "1.6.4",
     "verror": "1.6.0",
-    "wf": "1.2.0",
+    "wf": "git+https://github.com/joyent/node-workflow.git#fcbd66058d290fa8284a25bda854ff8743478b4f",
     "wf-moray-backend": "1.2.0"
   },
   "devDependencies": {
-- 
2.21.0

